float32 mean
float32 median
float32 variance
float32 std_dev